packages/core/src/components/steps-modal/steps-modal.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  forceUpdate,
+  h,
+} from '@stencil/core'
+
+@Component({
+  tag: 'atom-steps-modal',
+  styleUrl: 'steps-modal.scss',
+  shadow: false,
+})
+export class AtomStepsModal {
+  @Prop() steps: number = 0
+  @Prop({ mutable: true }) currentStep: number = 0
```

Comment:
if you set a parameter, you don't need to add type
```suggestion
  @Prop({ mutable: true }) currentStep = 0
```

Reply:
done at [693539c](https://github.com/juntossomosmais/atomium/pull/599/commits/693539c1609736fc4c7eaffbc565d0631bc03425)